lib/jsonld-signatures.js

```diff
@@ -32,6 +66,44 @@ api.sign = async function sign(document, {
   }
 };
 
+/**
+ * Verifies the linked data signature on the provided document.
+ *
+ * @param {object|string} document - The document with one or more proofs to be
+ *   verified, either a string URL (resolved to an object via the given
+ *   `documentLoader`) or a plain object (JSON-LD document).
+ *
+ * @param {LinkedDataSignature|LinkedDataSignature[]} suite -
+ *   Acceptable signature suite instances for verifying the proof(s).
+ *
+ * @param {ProofPurpose} purpose - A proof purpose instance that will
+ *   match proofs to be verified and ensure they were created according to
+ *   the appropriate purpose.
+ *
+ * Advanced optional parameters and overrides:
+ *
+ * @param {function} [documentLoader]  - A custom document loader,
+ *   `Promise<RemoteDocument> documentLoader(url)`.
+ * @param {function} [expansionMap] - A custom expansion map that is
+ *   passed to the JSON-LD processor; by default a function that will throw
+ *   an error when unmapped properties are detected in the input, use `false`
+ *   to turn this off and allow unmapped properties to be dropped or use a
+ *   custom function.
+ * @param {boolean} [compactProof=true] - Indicates that this method cannot
+ *   assume that the incoming document has defined all proof terms in the
+ *   same way as the `constants.SECURITY_CONTEXT_URL` JSON-LD `@context`.
+ *   This means that this method must compact any found proofs to this
+ *   context for internal and extension processing; this is the default
+ *   behavior. To override this behavior and optimize away this step because
+ *   the caller knows that the input document's JSON-LD `@context` defines
+ *   the proof terms in the same way, set this flag to `false`.
+ *
+ * @return {Promise<{verified: boolean, results: Array, error: *}>} resolves
```

Comment:
@dmitrizagidulin this needs to be updated to latest API.

Reply:
@mattcollier What's the latest API? (I just double checked, and the signature seems to be the same?)

Reply:
See: https://github.com/digitalbazaar/jsonld-signatures/blob/master/CHANGELOG.md#changed-1

Reply:
@mattcollier Ahhh thank you!

Reply:
Should be updated now.